spec/components/check_answers_summary_list_card_component_spec.rb

```diff
@@ -5,103 +5,68 @@
+  end
 
-        let(:log) { build(:lettings_log, :in_progress, age2: 99, startdate: Time.zone.local(2021, 5, 1)) }
+  context "when log was imported with a bulk upload creation method, without bulk upload id and has an unanswered question" do
+    let(:log) { create(:lettings_log, :in_progress, creation_method: "bulk upload", age2: 99, bulk_upload_id: nil) }
 
-        it "displays normal copy with muted colour " do
-          expect(rendered).to have_selector("span", class: "app-!-colour-muted", text: "You didn’t answer this question")
-        end
-      end
+    it "displays tweaked copy in red" do
+      expect(rendered).not_to have_selector("span", class: "app-!-colour-red", text: "You still need to answer this question")
+    end
+  end
 
-      context "when log was created via a bulk upload and has an unanswered optional question" do
-        subject(:component) { described_class.new(questions:, log:, user:) }
+  context "when log was not created via a bulk upload and has an unanswered question" do
+    it "displays normal copy with muted colour " do
+      expect(rendered).to have_selector("span", class: "app-!-colour-muted", text: "You didn’t answer this question")
+    end
+  end
 
-        let(:subsection_id) { "setup" }
-        let(:bulk_upload) { create(:bulk_upload) }
-        let(:log) { build(:lettings_log, :completed, creation_method: "bulk upload", tenancycode: nil, startdate: Time.zone.local(2021, 5, 1), bulk_upload:) }
+  context "when log was created via a bulk upload and has an unanswered optional question" do
+    let(:subsection_id) { "setup" }
+    let(:bulk_upload) { create(:bulk_upload) }
+    let(:log) { create(:lettings_log, :completed, creation_method: "bulk upload", tenancycode: nil, bulk_upload:) }
 
-        it "displays tweaked copy in red" do
-          expect(rendered).to have_selector("span", class: "app-!-colour-muted", text: "You didn’t answer this question")
-          expect(rendered).not_to have_selector("span", class: "app-!-colour-red", text: "You still need to answer this question")
-        end
-      end
+    it "displays tweaked copy in red" do
+      expect(rendered).to have_selector("span", class: "app-!-colour-muted", text: "You didn’t answer this question")
+      expect(rendered).not_to have_selector("span", class: "app-!-colour-red", text: "You still need to answer this question")
     end
   end
 
-  context "when in 23/24 collection" do
+  context "when before 23/24 collection" do
     context "when given a set of questions" do
-      let(:user) { create(:user) }
-      let(:log) { create(:lettings_log, :completed, age2: 99, startdate: Time.zone.local(2023, 5, 1)) }
-      let(:subsection_id) { "household_characteristics" }
-      let(:subsection) { log.form.get_subsection(subsection_id) }
-      let(:questions) { subsection.applicable_questions(log) }
+      let(:log) { build(:lettings_log, :completed, age2: 99, startdate: Time.zone.local(2021, 5, 1), assigned_to: create(:user)) }
```

Comment:
I'm surprised it still makes sense to test a log with a start date so old.
But from the looks of the test it doesn't really matter.